src/main/java/TaskList.java

```diff
@@ -0,0 +1,49 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+public class TaskList {
+    public ArrayList<Task> taskList;
+    
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+    
+    public TaskList() {
+        taskList = new ArrayList<>();
+    }
+    
+    public int taskListLength() {
+        return taskList.size();
+    }
+    
+    public Task addTask(Task task) {
+        taskList.add(task);
+        return task;
+    }
+    
+    public Task deleteTask(int index) {
+        Task task = taskList.get(index);
+        taskList.remove(index);
+        return task;
+    }
+    
+    public ArrayList<Task> getTaskList() {
+        return taskList;
+    }
+
+    public void findTask(String display) {
+        ArrayList<String> matches = new ArrayList<>();
+        String task = display.substring(5);
```

Comment:
Like previous, is there any reason you avoid using a constant for your substring() method calls?